resources/i18n/vi.json

```diff
@@ -163,7 +163,7 @@
       "DescSingleNode": " <p> Khi chạy một phiên, nút được quản lý và các nút công nhân được đặt trên một nút vật lý hoặc máy ảo duy nhất. </p>",
       "DescMultiNode": " <p> Khi chạy một phiên, một nút được quản lý và một hoặc nhiều nút công nhân được chia thành nhiều nút vật lý hoặc máy ảo. </p>",
       "CPU": "CPU",
-      "Memory": "Ký ức",
+      "Memory": "Bộ nhớ",
```

Comment:
Both "Ký ức" and "Bộ nhớ" are translated as "memory" in English. Therefore, I inquired with ChatGPT about the distinctions between these two terms.

In Vietnamese, "Ký ức" and "Bộ nhớ" have distinct meanings:

Ký ức: This term refers to memory in the psychological or emotional sense, often related to personal experiences, recollections, or memories that individuals hold.

Bộ nhớ: This term specifically refers to memory in a technical context, particularly in computing. It denotes the hardware or storage used in computers and devices to store data temporarily or permanently.

So, for computer memory, "Bộ nhớ" is the correct term to use.